### PR TITLE
Payables page

### PR DESCRIPTION
**Pull request by konstantin1390**
Fixes #

### Changes

Summary of changes you made.

* Change 1
* Change 2

### Test plan

Explain how to test changes you made.

* [ ] Step 1
* [ ] Step 2

### Demo (optional)

Sreenshots (before & after) or video that show result of your work.

### CODE REVIEW

File: src/client/rewards/Payables/Payables.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+const Payables = ({ intl, userName, currentSteemDollarPrice, filterData, setPaymentUser }) => {
+  const payableFilters = {};
+  _.map(filterData, f => {
+    if (f.value === 15) {
+      payableFilters.days = 15;
+    }
+    if (f.value === 10) {
+      payableFilters.payable = 10;
+    }
```
Comment (milklim) [inline src/client/rewards/Payables/Payables.js]: check filters by filter name

File: src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js [hunk 2]
```diff
@@ -5,38 +5,78 @@ import React from 'react';
...
     <div key={`${key}-${filterName}`} className="RewardsFiltersPanel__item-wrap">
       <Checkbox onChange={() => setFilterValue(filterName, key)} checked={checked} />
       <div className="RewardsFiltersPanel__name">{filterName}</div>
     </div>
   );
 
+  const filterPayablesLayout = (obj, key, checked) => (
+    <div key={`${key}-${obj.filterName}`} className="RewardsFiltersPanel__item-wrap">
+      <Checkbox onChange={() => setPayablesFilterValue(obj, key)} checked={checked} />
+      <div className="RewardsFiltersPanel__name">{obj.filterName}</div>
```
Comment (milklim) [inline src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js]: add filter's name to locales

File: src/client/rewards/Payables/UserPayableCard/UserPayableCrad.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```
Comment (milklim) [inline src/client/rewards/Payables/UserPayableCard/UserPayableCrad.js]: rename component

File: src/client/rewards/Payment/Payment.less [hunk 4]
```diff
@@ -0,0 +1,35 @@
+@import (reference) '../../styles/modules/variables.less';
+@import (reference) '../../styles/custom.less';
+
+.Payment {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
```
Comment (milklim) [inline src/client/rewards/Payment/Payment.less]: use common layout of the app

File: src/client/rewards/Payment/PaymentTable/PaymentTable.less [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+  &__action-column {
+    text-align: start;
+  }
+
+  &__balance-column {
+    font-weight: bold !important;
+  }
+}
+
+.PaymentTable__action-column > div > span {
```
Comment (milklim) [inline src/client/rewards/Payment/PaymentTable/PaymentTable.less]: remove, use styles-shortcodes

File: src/client/rewards/Payment/PaymentTable/PaymentTableRow.js [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+              sponsorName: sponsor.sponsor,
+            },
+          )}
+        </div>
+        {sponsor && sponsor.details ? (
+          <React.Fragment>
+            <p>
+              {`-`}
+              {sponsor.details.main_object}
+            </p>
```
Comment (milklim) [inline src/client/rewards/Payment/PaymentTable/PaymentTableRow.js]: use unordered list

File: src/client/rewards/Rewards.js [hunk 7]
```diff
@@ -82,8 +81,10 @@ class Rewards extends React.Component {
     campaignsTypes: [],
     isModalDetailsOpen: false,
     objectDetails: {},
-    activeFilters: { guideNames: [], types: [] },
+    activeFilters: { guideNames: [], types: [], payables: [] },
```
Comment (milklim) [inline src/client/rewards/Rewards.js]: remove payables property

File: src/client/rewards/Rewards.js [hunk 8]
```diff
@@ -131,6 +134,19 @@ class Rewards extends React.Component {
...
+    const activeFilters = [...this.state.activePayableFilters];
+    if (_.find(activeFilters, ['filterName', filter.filterName])) {
+      this.setState({
+        activePayableFilters: activeFilters.filter(f => f.filterName !== filter.filterName),
+      });
+    } else {
+      activeFilters.push(filter);
+      this.setState({ activePayableFilters: activeFilters });
+    }
+    this.setState({ loadingCampaigns: true });
```
Comment (milklim) [inline src/client/rewards/Rewards.js]: remove line

File: src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js [hunk 9]
```diff
@@ -49,6 +89,9 @@ RewardsFiltersPanel.propTypes = {
   activeFilters: PropTypes.shape().isRequired,
   intl: PropTypes.shape().isRequired,
   setFilterValue: PropTypes.func.isRequired,
+  location: PropTypes.shape().isRequired,
+  setPayablesFilterValue: PropTypes.func.isRequired,
+  activePayableFilters: PropTypes.arrayOf(PropTypes.shape()).isRequired,
   // intl: PropTypes.shape().isRequired,
```
Comment (milklim) [inline src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js]: remove comment

File: src/client/rewards/Rewards.js (hunk 7 above)
Comment (konstantin1390) [reply src/client/rewards/Rewards.js]: done

File: src/client/rewards/Payables/UserPayableCard/UserPayableCrad.js (hunk 3 above)
Comment (konstantin1390) [reply src/client/rewards/Payables/UserPayableCard/UserPayableCrad.js]: done

File: src/client/rewards/Rewards.js (hunk 8 above)
Comment (konstantin1390) [reply src/client/rewards/Rewards.js]: done

File: src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js (hunk 9 above)
Comment (konstantin1390) [reply src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js]: done

File: src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js (hunk 2 above)
Comment (konstantin1390) [reply src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js]: done

File: src/client/rewards/Payables/Payables.js (hunk 1 above)
Comment (konstantin1390) [reply src/client/rewards/Payables/Payables.js]: done

File: src/client/rewards/Payment/PaymentTable/PaymentTable.less (hunk 5 above)
Comment (konstantin1390) [reply src/client/rewards/Payment/PaymentTable/PaymentTable.less]: done

File: src/client/rewards/Payment/PaymentTable/PaymentTableRow.js (hunk 6 above)
Comment (konstantin1390) [reply src/client/rewards/Payment/PaymentTable/PaymentTableRow.js]: done

File: src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js [hunk 10]
```diff
@@ -22,17 +22,23 @@ const RewardsFiltersPanel = ({
...
-    <div key={`${key}-${obj.filterName}`} className="RewardsFiltersPanel__item-wrap">
-      <Checkbox onChange={() => setPayablesFilterValue(obj, key)} checked={checked} />
-      <div className="RewardsFiltersPanel__name">{obj.filterName}</div>
+  const filterPaymentLayout = (obj, checked) => (
+    <div key={`${obj.filterName}`} className="RewardsFiltersPanel__item-wrap">
+      <Checkbox onChange={() => setPayablesFilterValue(obj)} checked={checked} />
+      <div className="RewardsFiltersPanel__name">
+        {intl.formatMessage(
+          { id: `filter_${obj.filterName}`, defaultMessage: obj.defaultMessage },
+          { days: 15, payable: 10 },
```
Comment (milklim) [inline src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js]: use obj.value

File: src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js [hunk 11]
```diff
@@ -70,9 +76,8 @@ const RewardsFiltersPanel = ({
               })}:`}
             </div>
             {_.map(payablesFilterData, payable =>
-              filterPayablesLayout(
+              filterPaymentLayout(
                 payable,
-                'payables',
                 !!_.find(activePayableFilters, ['filterName', payable.filterName]),
```
Comment (milklim) [inline src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js]: try this: activePayableFilters.some(f => f.filterName === payable.filterName)

File: src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js (hunk 10 above)
Comment (konstantin1390) [reply src/client/rewards/RewardsFiltersPanel/RewardsFiltersPanel.js]: done

**Pull request merged**